lib/studies/fortnightly/app.dart

```diff
@@ -24,6 +24,7 @@ class FortnightlyApp extends StatelessWidget {
         ? const _FortnightlyHomeDesktop()
         : const _FortnightlyHomeMobile();
     return MaterialApp(
+      restorationScopeId: 'appNavigator',
```

Comment:
Should the ID be something like `fortnightlyApp`?

Reply:
It shouldn't matter since only one mini-app can be open at any given time so there won't be a conflict. I'll change it anyway since it's pretty quick and may be clearer.